app/views/foreman_virt_who_configure/configs/show.html.erb

```diff
@@ -1,61 +1,91 @@
 <% stylesheet 'foreman_virt_who_configure/config' %>
 <% javascript 'foreman_virt_who_configure/config_copy_and_paste' %>
+<% title(_("Virt-who Configuration %s") % @config.name) %>
+<%= title_actions(
+        display_link_if_authorized(_('Edit'), hash_for_edit_foreman_virt_who_configure_config_path(@config).merge(:permission => 'edit_virt_who_config', :auth_object => @config), :class => 'btn btn-default'),
+        display_delete_if_authorized(hash_for_foreman_virt_who_configure_config_path(@config).merge(:permission => 'destroy_virt_who_config', :auth_object => @config), :class => 'btn btn-default', :data => {:confirm => _("Delete virt-who configuration %s?") % config.name})
+    ) %>
 
-<div class="clearfix">
-  <div class="form-group">
-    <div class="row">
-      <div class="col-md-12">
-        <h1><%= _("Virt-who Configuration %s") % @config.name %></h1>
-      </div>
-      <div class="col-md-6">
-        <%= _("Use either hammer command or the script below to deploy this configuration. Both require root privileges.  Run one of them on the target host which has access to Red Hat Satellite Tools repository and will run virt-who reporting, preferably Satellite host:") %>
-      </div>
-      <div class="col-md-2">
-        <div class="pull-right">
-          <p>
-            <%= display_link_if_authorized _('Edit this configuration'), hash_for_edit_foreman_virt_who_configure_config_path(@config).merge(:permission => 'show_virt_who_config', :auth_object => @config), :class => 'btn btn-default' %>
-          </p>
+<ul id="config-tab" class="nav nav-tabs">
+  <li class="active"><a href="#overview" data-toggle="tab"><%= _('Overview') %></a></li>
+  <li><a href="#deploy" data-toggle="tab"><%= _('Deploy') %></a></li>
+</ul>
+
+<div id="config-tab-content" class="config-tab-content tab-content">
+  <div class="tab-pane active in" id="overview">
+    <div class="clearfix">
+      <div class="form-group">
+        <div class="col-md-12">
+          <div class="row">
+            <h3><%= _('Details') %></h3>
```

Comment:
Headings on the other tab are all `h2`. Could you change this one to `h2` too? They're on the same level and it feels inconsistent when I'm switching tabs. What do you think?

Reply:
I actually replaced the other tab to use h3 to be more consistent with smart proxy page, h2 we left overs from previous page design

Reply:
Ok, as long as it's consistent it works for me.